### ISSUE TITLE
selector-type-no-unknown

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1254]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bad
```css
htmlelement {
  color: red;
}
```

God:
```css
section {
  color: red;
}
```

Search and include html library, which can validate html tags

**Comment by evilebottnawi** [comment]
@davidtheclark what do you think about this package https://github.com/sindresorhus/html-tags?

**Comment by davidtheclark** [comment]
Looks great to me.

**Comment by mehcode** [comment]
@evilebottnawi It doesn't look like that package includes SVG tags (which are valid targets for CSS). How about supplementing with https://github.com/element-io/svg-tags ?

**Comment by evilebottnawi** [comment]
@mehcode thanks

**Comment by evilebottnawi** [comment]
Done in https://github.com/stylelint/stylelint/pull/1262
